src/Components/test/E2ETest/Tests/SignalRClientTest.cs

```diff
@@ -37,11 +38,15 @@ protected override void InitializeAsyncCore()
             Browser.Exists(By.Id("signalr-client"));
         }
 
-        [Fact]
-        public void SignalRClientWorks()
+        [Theory]
+        [InlineData(HttpTransportType.WebSockets)]
+        [InlineData(HttpTransportType.LongPolling)]
+        public void SignalRClientWorks(HttpTransportType transportType)
```

Comment:
Could we make this two separate tests?

Reply:
What for?

Reply:
Individual tests are much more easier to work with than theory tests. Easier to individually skip, search for, filter in dotnet test etc.